stdlib/LinearAlgebra/test/lu.jl

```diff
@@ -341,4 +337,13 @@ end
     @test F.p == []
 end
 
+@testset "Issue #33547, condition number of 2x2 matrix" begin
+    M = [1.0 -2.0; -2.0 -1.5]
+    @test cond(M, 1) ≈ 2.227272727272727
+end
+@testset "Issue #33297, condition number of singular matrix" begin
+    M = [1. 1.; 0. 0.]
+    @test cond(M, 1) == Inf
+end
+
```

Comment:
Maybe we could move these tests to `dense.jl`? The current implementation of `cond` has nothing in common with `lu` anymore and resides in `dense.jl`. So doesn't it make more sense to have these tests also in `dense.jl`?